### PR TITLE
More Klocwork fixes

### PR DESCRIPTION
**Pull request by strassek**

### CODE REVIEW

File: wsi/drm/drmdisplay.cpp [hunk 1]
```diff
@@ -135,11 +135,13 @@ bool DrmDisplay::ConnectDisplay(const drmModeModeInfo &mode_info,
   SetPowerMode(power_mode_);
 
   // This is a valid case on DSI panels.
-  if (!broadcastrgb_props)
+  if (broadcastrgb_props == NULL)
```
Comment (MicheleL-Intel) [inline wsi/drm/drmdisplay.cpp]: Print error message here so that we're aware of problems during runtime.

File: wsi/drm/drmdisplay.cpp [hunk 2]
```diff
@@ -135,11 +135,13 @@ bool DrmDisplay::ConnectDisplay(const drmModeModeInfo &mode_info,
...
     return true;
 
-  if (!(broadcastrgb_props->flags & DRM_MODE_PROP_ENUM))
+  if (!(broadcastrgb_props->flags & DRM_MODE_PROP_ENUM)) {
+    drmModeFreeProperty(broadcastrgb_props);
```
Comment (MicheleL-Intel) [inline wsi/drm/drmdisplay.cpp]: Good catch but just a minor nit - this is not a NULL pointer dereference issue.  Was this reported by Klockwork?

File: wsi/drm/drmplane.cpp [hunk 3]
```diff
@@ -252,6 +252,8 @@ bool DrmPlane::Initialize(uint32_t gpu_fd,
 
       formats_modifiers_.emplace_back(modifiers_obj);
     }
+
+    drmModeFreePropertyBlob(blob);
```
Comment (MicheleL-Intel) [inline wsi/drm/drmplane.cpp]: Same comment as 'drmModeFreeProperty(broadcastrgb_props)' - this is not a NULL pointer dereference issue.  Was this reported by Klockwork?

File: wsi/drm/drmdisplay.cpp (hunk 1 above)
Comment (strassek) [reply wsi/drm/drmdisplay.cpp]: Based on the comment above, I have to assume that this isn't an error path. Maybe an info print would be suitable.

File: wsi/drm/drmdisplay.cpp (hunk 2 above)
Comment (strassek) [reply wsi/drm/drmdisplay.cpp]: Klocwork caught this as a possible memory leak

File: wsi/drm/drmplane.cpp (hunk 3 above)
Comment (strassek) [reply wsi/drm/drmplane.cpp]: Klocwork caught this as a possible memory leak

**Comment by strassek** [comment]
@MicheleL-Intel Added a warning to the case where we return early from ConnectDisplay()

**Comment by MicheleL-Intel** [comment]
LGTM.

**Pull request merged**